src/pages/index.js

```diff
@@ -115,12 +117,15 @@ profileEditBtn.addEventListener("click", () => {
 const profilePictureModal = new PopupWithForm(
   "#profile-picture-edit-modal",
   () => {
+    const values = profilePictureModal.getInputValues();
+
     profilePictureModal.loadingButtonState();
     api
-      .setProfilePicture(profilePictureFormInput.value)
+      .setProfilePicture(values.url)
       .then((res) => {
-        profileAvatar.src = res.avatar;
+        userInfo.setUserAvatar(res.avatar);
         profilePictureModal.close();
+        profilePictureSubmitBtn.classList.add("modal__button_disabled");
```

Comment:
Now only class `FormValidator` should **`disable/enable`** submit buttons and remove errors from the inputs (if you do it)